torchrl/modules/tensordict_module/probabilistic.py

```diff
@@ -5,29 +5,27 @@
 
 from typing import Optional, Sequence, Type, Union
 
-from tensordict.nn import (
-    ProbabilisticTensorDictModule as _ProbabilisticTensorDictModule,
-)
+from tensordict.nn import ProbabilisticTensorDictModule
 
 from torchrl.data import TensorSpec
 from torchrl.modules.distributions import Delta
-from torchrl.modules.tensordict_module.common import TensorDictModule
+from torchrl.modules.tensordict_module.common import SafeModule
 
 
-class ProbabilisticTensorDictModule(_ProbabilisticTensorDictModule, TensorDictModule):
+class SafeProbabilisticModule(ProbabilisticTensorDictModule, SafeModule):
     """A probabilistic TD Module.
```

Comment:
`A :obj:``SafeProbabilisticModule`` is an :obj:``tensordict.nn.ProbabilisticTensorDictModule`` subclass that accepts a :obj:``TensorSpec`` as argument to control the output domain.`